src/admin.sol

```diff
@@ -67,6 +73,11 @@ contract Admin {
         return loan;
     }
 
+    function file(bytes32 what, address addr) public auth {
```

Comment:
if we call it `depend` we should also move this method into the new `depend`.

The idea is to use `depend` for contract dependencies and `file` for other external input like a
principal or a nft etc.

Reply:
actually, i was thinking about it as core contracts v external (substitutable) module contracts (appraiser being substituted eventually by potentially underwriter system, etc). but we could also put all contract dependencies into `depend` and `file` just for external 'data' input. not opinionated on this.

Reply:
ok. maybe let's use `depend` for all contract dependencies for now.